neonvm/controllers/virtualmachine_controller.go

```diff
@@ -319,6 +324,11 @@ func (r *VirtualMachineReconciler) doReconcile(ctx context.Context, virtualmachi
 		meta.SetStatusCondition(&virtualmachine.Status.Conditions, metav1.Condition{Type: typeAvailableVirtualMachine, Status: metav1.ConditionUnknown, Reason: "Reconciling", Message: "Starting reconciliation"})
 	}
 
+	// Generate ssh secret name
+	if len(virtualmachine.Status.SSHSecretName) == 0 {
+		virtualmachine.Status.SSHSecretName = virtualmachine.Name
```

Comment:
Maybe you need to generate smth like `fmt.Sprintf("ssh-%s", virtualmachine.Name)`?

Reply:
I felt that might be redundant since secrets type are `kubernetes.io/ssh-auth`. For example if you run `kubectl get secrets`, the output looks like this:
```
NAME                   TYPE                     DATA   AGE
example                kubernetes.io/ssh-auth   2      17h
postgres14-disk-test   kubernetes.io/ssh-auth   2      17h
```

Reply:
In that case, maybe you don't need a separate `SSHSecretName` variable, and you can just use `virtualmachine.Name` directly?

Reply:
I am just saying, if there is no way `SSHSecretName` can ever be different, I'd not create another variable. Instead it can be made explicit by using setter and getter for the keys. 

But that's just ideas, I am fine with any way forward.

Reply:
It's more than the codebase. I want to make the dependency explicit on the k8s object such that we can see it during operation as well.

Reply:
Can you explain this a bit in details?

Reply:
Let's say you're investigating a VM object and run `kubectl get neonvm vmname -o yaml`. Now in the status part you'll see the `sshSecretName` and the `podName`. The dependencies here are clear.

Reply:
Changed in to `ssh-neonvm-<name>`: 6383f4e7a2e9a73569799ee85a8dc667358afcd4